src/js/stores/__tests__/MarathonStore-test.js

```diff
@@ -176,4 +177,14 @@ describe('MarathonStore', function () {
 
   });
 
+  describe('#processMarathonDeployments', function () {
```

Comment:
Usually we test all the actions that the store receives. Here is a good example to look at:
https://github.com/dcos/dcos-ui/blob/master/src/js/stores/__tests__/UserStore-test.js